.github/workflows/build.yml

```diff
@@ -0,0 +1,65 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Lint + Build + Test
```

Comment:
Could we rename this to `SE Soc CI` it looks like github shows the different stages /Lint /Build /Test already